src/components/common/link/link.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Linking, TouchableOpacity } from 'react-native';
+
+type Props = {
+    link: string;
+}
+
+const Link: React.FC<Props> = ({
+    children,
+    link,
+}) => {
+
+    const handleClick = () => {
+        Linking.canOpenURL(link).then(supported => {
+            if (supported) {
+                Linking.openURL(link);
+            } else {
+                console.log("Don't know how to open URI: " + link);
+            }
+        });
+    };
+
+    return (
+        <TouchableOpacity onPress={handleClick}>{children}</TouchableOpacity>
```

Comment:
No need for the children support here. Title should be fine.

Reply:
I thought it can be like a wrapper for buttons